### DISCUSSION TITLE
ChatGPTBrowserClient

### DISCUSSION THREAD

**Discussion by userluoc** [waylaidwanderer/node-chatgpt-api#102] [Help] [answered]
Is it necessary to use ChatGPTBrowserClient cookies? If so, how to obtain it?

**Comment by waylaidwanderer** [comment, marked as answer]
No, just the access token is enough if you use the reverse proxy. If it wasn't for Cloudflare being so strict then you'd be able to use the cookies without a reverse proxy, but that's not possible at the moment.

**Reply by userluoc** [reply]
I used the reverseProxyUrl you gave me: ' https://chatgpt.duti.tech/api/conversation ', do you need to change it?

**Reply by waylaidwanderer** [reply]
No, that one should be working fine.

**Reply by userluoc** [reply]
But I can't connect with the proxy addres you gave me https://chatgpt.duti.tech/api/conversation.  What's the matter? My token is absolutely correct. I don't set cookies

**Reply by waylaidwanderer** [reply]
It's hard to say what the issue is since it's working for most people, including me. What version of Node are you using? Maybe you can email me your session token so I can test it on my end...
